packages/devextreme/js/__internal/grids/new/grid_core/search/controller.test.ts

```diff
@@ -26,7 +26,7 @@ describe('Search', () => {
       jest.clearAllMocks();
     });
 
-    it('should have highlightTextOptions$ from widget options', () => {
+    it('should have highlightTextOptions observable from widget options', () => {
```

Comment:
minor
let's fix describes in this file
describe('SearchController') -> describe('highlightTextOptions') -> it('should have value from options')